src/main/java/com/github/havardh/javaflow/plugins/JavaflowMojo.java

```diff
@@ -93,7 +99,7 @@ private void run(Api api) {
             new InheritanceTransformer(),
             new SortedTypeTransformer()
         ),
-        singletonList(new MemberFieldsPresentVerifier(typeMap)),
+        getAllVerifiers(api.getVerifications(), typeMap),
         new FlowWriter(converter),
         asList(
             new CommentPrependTransformer("Generated by javaflow 1.3.0"),
```

Comment:
Yes I know this is kind of tedious, but could you update the version to 1.4.0 :)